### ISSUE TITLE
Photon should have a docker sub package that provides the docker buildx plugin

### ISSUE THREAD

**Issue by dcasota** [vmware/photon#1547]
### Is your feature request related to a problem? Please describe.

photon os installer' `create-image-util` fails with docker 24.05 because of an known issue "unknown flag: --build-context".

### Describe the solution you'd like

`tdnf install docker-buildx`

### Additional context

see https://github.com/vmware/photon-os-installer/issues/26

**Comment by oliverkurth** [comment]
Thanks for reporting. I created an internal issue for this.

**Comment by dcasota** [comment]
no hurry. Facing the amount of cve which have been released for moby 24.x, 25.x and related to buildkit, testing across boundaries is crucial.
0.13.1 with 24.05 works. Facing the 24.05-26.0 releasenotes so far, escaping containers is a big issue. Good luck.

**Comment by dcasota** [comment]
See https://github.com/vmware/photon/commit/74cb2c22655d7d88d4f2191c7d029d1b354aa5cf

**Comment by dcasota** [comment]
Hi @ssheedi, please could you double check the bits? Actually there seems a file checksum issue. Thank you for your help.

![image](https://github.com/user-attachments/assets/8c757c0d-40e3-47fa-a4e1-a8b0b71ccaae)

**Comment by dcasota** [comment]
docker-buildx on Ph5 x86_64 works perfectly. Many thanks! Please, push it to Ph4.0 as well.

**Comment by YustasSwamp** [comment]
@sshedi ^

**Comment by sshedi** [comment]
Please with this command and give your observations: `sudo tdnf install --refresh -y docker-buildx`
Probably your repodata is not in sync with server, hence the failure.

**Comment by dcasota** [comment]
Yes, but unfortunately there is no docker-buildx package on the on the open-source 4.0 branch in comparison to the 5.0 branch. aybe there is a reason on this? I don't know.

![Image](https://github.com/user-attachments/assets/737cdab8-6675-480f-ac3a-73c4cb883f0d)

In my test lab, a newly compiled 4.0 ISO confirms the situation.

![Image](https://github.com/user-attachments/assets/62848f48-f941-41b7-99db-7591350f5c7f)

Low priority. I will recheck in a few weeks again.

**Comment by sshedi** [comment]
> docker-buildx on Ph5 x86_64 works perfectly. Many thanks! Please, push it to Ph4.0 as well.

Sorry, I missed this message. I will add it.

**Comment by dcasota** [comment]
works

fyi
tested on
- Ph4 x86_64: 0.17.1-1
- Ph5 x86_64 + aarch64: 0.17.1-3
